src/Spice86.Core/Emulator/InterruptHandlers/Bios/SystemBiosInt15Handler.cs

```diff
@@ -20,14 +23,25 @@ public class SystemBiosInt15Handler : InterruptHandler {
     }
 
     /// <summary>
-    /// No extended memory size present. Yet.
-    /// Reports 0 in AX.
+    /// Reports extended memory size in AX.
     /// </summary>
     public void GetExtendedMemorySize() {
-        //We've got no extended memory (yet)
         _state.AX = 0;
     }
 
+    public void CopyExtendedMemory(bool calledFromVm) {
+        bool enabled = _memory.IsA20Enabled;
+        _machine.Memory.EnableOrDisableA20Gate(true);
```

Comment:
why not _memory.IsA20Enabled = true (seems to do the same thing)?

Reply:
Properties are not supposed to have any side effect (calls, calculations, touching other properties, ...).
This is supposed to modify the wrapping, so this is why it is a method.
It's not implemented, so I'll remove it.

Reply:
Code removed.